{"identifier":{"url":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"sections":[],"topicSections":[{"identifiers":["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/failureReason","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/helpAnchor","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/recoverySuggestion"],"generated":true,"title":"Instance Properties"}],"variants":[{"paths":["\/documentation\/postgresclientorm\/tableobjecterror\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"article","metadata":{"modules":[{"name":"PostgresClientORM"}],"title":"LocalizedError Implementations","role":"collectionGroup"},"schemaVersion":{"major":0,"patch":0,"minor":3},"hierarchy":{"paths":[["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError"]]},"references":{"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/recoverySuggestion":{"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/recoverySuggestion","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"type":"topic","kind":"symbol","url":"\/documentation\/postgresclientorm\/tableobjecterror\/recoverysuggestion","title":"recoverySuggestion","abstract":[],"role":"symbol"},"doc://PostgresClientORM/documentation/PostgresClientORM":{"abstract":[],"kind":"symbol","url":"\/documentation\/postgresclientorm","type":"topic","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","title":"PostgresClientORM","role":"collection"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/failureReason":{"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/failureReason","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"type":"topic","kind":"symbol","url":"\/documentation\/postgresclientorm\/tableobjecterror\/failurereason","title":"failureReason","abstract":[],"role":"symbol"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError":{"type":"topic","title":"TableObjectError","url":"\/documentation\/postgresclientorm\/tableobjecterror","abstract":[],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"TableObjectError","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError","navigatorTitle":[{"text":"TableObjectError","kind":"identifier"}],"role":"symbol"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/helpAnchor":{"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/helpAnchor","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"type":"topic","kind":"symbol","title":"helpAnchor","url":"\/documentation\/postgresclientorm\/tableobjecterror\/helpanchor","abstract":[],"role":"symbol"}}}